firmware/controllers/trigger/trigger_decoder.cpp

```diff
@@ -377,6 +377,11 @@ void TriggerCentral::validateCamVvtCounters() {
 }
 
 bool TriggerState::syncSymmetricalCrank(int mod, int remainder) {
+	// In case this engine has the same "length" of primary trigger and VVT, we're already sync'd
+	if (mod == remainder) {
```

Comment:
I am failing to come up with a scenario where mod == remainder, can you please help?

I am concerned about mixing too many unrelated changes under one ticket

Reply:
#3018